patches/rrweb@2.0.0-alpha.13.patch

```diff
@@ -205,8 +205,8 @@ index da2c103fe6b1408a5996f0eb3bf047571e434cc2..745c0bee7ca8d49d5c3f4102d71fe965
 +
 +                // createImageBitmap throws if resizing to 0
 +                // Fallback to intrinsic size if canvas has not yet rendered
-+                const width = canvas.clientWidth || client.width;
-+                const height = canvas.clientHeight || client.height;
++                const width = canvas.clientWidth || canvas.width;
++                const height = canvas.clientHeight || canvas.height;

```

Comment:
@daibhin the check that returns early if `width` and/or `height` null/0 are gone, this can lead to the `createImageBitmap` throw again.
why did we remove it from the patch? or is it guaranteed that `canvas.width` and `canvas.height` are always non zero?

Reply:
Removed it because the version of rrweb we're on supports your change: https://github.com/rrweb-io/rrweb/blob/f3cf0928df30d5ed5c0d573c524be6e744c0f8d3/packages/rrweb/src/record/observers/canvas/canvas-manager.ts

(a little further up in the same file so you don't see it in this patch)

I confirmed it was still there in the `node_modules`:
<img width="683" alt="Screenshot 2024-06-04 at 15 25 22" src="https://github.com/PostHog/posthog-js/assets/6685876/dfcefcaa-412f-49d4-b144-5d2f601f488f">